src/main/java/de/tum/cit/aet/artemis/assessment/service/ResultService.java

```diff
@@ -692,4 +692,19 @@ public void deleteLongFeedback(List<Feedback> feedbackList, Result result) {
         List<Feedback> feedbacks = new ArrayList<>(feedbackList);
         result.updateAllFeedbackItems(feedbacks, true);
     }
+
+    /**
+     * Retrieves the number of students affected by a specific feedback detail text for a given exercise.
+     * <p>
+     * This method queries the repository to count the distinct students whose submissions were impacted
+     * by feedback entries matching the provided detail text within the specified exercise.
+     * </p>
+     *
+     * @param exerciseId the ID of the exercise for which the affected student count is requested.
+     * @param detailText the feedback detail text used to filter affected students.
+     * @return the total number of distinct students affected by the feedback detail text.
+     */
```

Comment:
I think we should shorten the comment, in its current state I also think it will not be maintained properly and might become misleading for future developers

We could also keep the param description a little shorter

```suggestion
    /**
     * Retrieves the number of students affected by a specific feedback detail text for a given exercise.
     *
     * @param exerciseId for which the affected student count is requested.
     * @param detailText used to filter affected students.
     * @return the total number of distinct students affected by the feedback detail text.
     */
```